Packages/com.unity.inputsystem/InputSystem/Editor/EditorInputControlLayoutCache.cs

```diff
@@ -135,7 +135,7 @@ public static IEnumerable<OptionalControl> GetOptionalControlsForLayout(string l
         public static Texture2D GetIconForLayout(string layoutName)
         {
             if (string.IsNullOrEmpty(layoutName))
-                throw new ArgumentNullException(nameof(layoutName));
+                return null;
```

Comment:
Why?

Reply:
Woops, that was to bypass an error temporarily.
Will investigate the error before comitting.